Menu/Matcher/Voter/ChildrenVoter.php

```diff
@@ -1,61 +0,0 @@
-<?php
```

Comment:
This is a BC break

Reply:
this special voter is breaking the concept of `current` item for people using KnpMenu in their project outside the admin part. So removing the voter actually fixes bugs.

Reply:
I think I'll keep the class around but remove the service, and deprecate the class.